server/routes/ad.ts

```diff
@@ -302,10 +303,7 @@ const searchDetector = async (
       },
     };
   } catch (err) {
-    if (
-      err.statusCode === 404 &&
-      get<string>(err, 'body.error.type', '') === 'index_not_found_exception'
-    ) {
+    if (isIndexNotFoundError(err)) {
       return { ok: true, response: { totalDetectors: 0, detectors: [] } };
     }
     console.log('Anomaly detector - Unable to search detectors', err);
```

Comment:
nit: would prefer logging before any `return`, right after `catch`. same below.

Reply:
> nit: would prefer logging before any `return`, right after `catch`. same below.

In this case, do you mean adding an additional log in the `if` block here? Could add

Reply:
no. just moving `console.log()` before `if(isIndexNotFoundError)`should be good.

Reply:
yeah I see, will change